apps/condo/domains/notification/schema/helpers.js

```diff
@@ -0,0 +1,20 @@
+const faker = require('faker')
+const sample = require('lodash/sample')
+
+const { PUSH_TRANSPORT_TYPES } = require('@condo/domains/notification/constants/constants')
+
+const getRandomTokenData = (extraAttrs = {}) => {
```

Comment:
Please, move this to `testHelpers` folder, since It's used only for tests.

Reply:
`testSchema`

Reply:
Use `buildFakeAddressAndMeta` as example

Reply:
moved to utils/testSchema